rabbitmq/json-overview.yml

```diff
@@ -1,180 +1,102 @@
 type: json
 data:
   url: /api/overview
-  server: ${SPM_MONITOR_RABBITMQ_API_URL}
-  basicHttpAuthUsername: ${SPM_MONITOR_RABBITMQ_BASICAUTH_USERNAME}
-  basicHttpAuthPassword: ${SPM_MONITOR_RABBITMQ_BASICAUTH_PASSWORD}
+  server: ${RABBITMQ_API_URL}
+  basicHttpAuthUsername: ${RABBITMQ_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${RABBITMQ_BASICAUTH_PASSWORD}
 
 observation:
   - name: overview
     metricNamespace: rabbitmq
     path: $.
 
     metric:
-      - name: object_totals.connections
+      - name: object.totals.connections
```

Comment:
Probably should be `connections.total` or `objects.connections.totals` (only if we really want to keep notion on `object` in metric name). Same goes for other metrics below.